growth/too/tasks/tiles.py

```diff
@@ -309,20 +326,13 @@ def get_planned_observations(
                         },
                         'properties': {
                             'telescope': telescope,
-                            'field_id': int(field_id),
+                            'field_id': int(field.field_id),
```

Comment:
We decided that the field ID did not have to be filled in here.

Reply:
I think we decided it would be ok to not have it displayed in the tooltip, but this got it working, so no downside.

Reply:
However, adding the field ID requires three database transactions instead of one, since you have to look up the field ID.

In any case, the field ID is not meaningful for galaxy fields.

Reply:
That isn't true right? Each of the galaxies (as they are added to the catalog) will have its own row?

Reply:
I'm sorry, what isn't true?

Reply:
"the field ID is not meaningful for galaxy fields" -> Once it's in the database, the galaxy has a unique field_id.

Reply:
It has a field_id, but why would the user care what it is?

Reply:
Well once it is pinned to something, those observations of the same galaxies will end up with the same field_id. I am not saying I use it now, but it seems silly to not have it noted correctly when we trivially can. If you feel strongly enough that its useless, please go ahead and remove it and commit it, I would like to move on.

Reply:
I was going to also remark that I don't see any particular need to reuse existing galaxy fields. It adds a lot of extra database queries.

Reply:
Ok I removed the contour call, hopefully that does it.

Reply:
No, we definitely need the contour column filled in so that we can plot the field.

Reply:
ok pushed in latest version.